kbchat/kbchat.go

```diff
@@ -133,6 +135,13 @@ func (a *API) startPipes() (err error) {
 		a.apiCmd.Process.Kill()
 	}
 	a.apiCmd = nil
+
+	if a.runOpts.StartService {
+		go func() {
+			a.runOpts.Command("service").Run()
```

Comment:
Not sure if there's a better way to spin up the service in Go

Reply:
I think here it would be better to use [`Cmd.Start()`](https://golang.org/pkg/os/exec/#Cmd.Start) without the goroutine. 

I think it also raises the question of if/how one should check for errors here. I think it is probably fine as is but it would be nice if it listened for a line of the form `net.Listen on unix:/tmp/asdyhausdyuqywedu/.config/keybase/keybased.sock` (via StderrPipe and StdoutPipe) in order to check that Keybase starts correctly, but not a big deal.

Reply:
Agreed, but I think we should do this in another PR just to keep this from getting too big